src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -154,45 +145,213 @@ export const flattenNmKResultsConstraints = (
     return rows;
 };
 
+const makeColumn = ({
+    headerName,
+    field = '',
+    valueGetter,
+    cellRenderer,
+    isSortable = false,
+    isHidden = false,
+    isFilterable = false,
+    filterParams,
+    filtersDef,
+    filterSelector,
+    sortConfig,
+    valueFormatter,
+    onSortChanged,
+    updateFilter,
+}: CustomColDef) => {
```

Comment:
this method has a lot of params which makes it difficult to know how to reuse, any way to make it more simple ?

Reply:
I understand to make it a utility function rather than a useCallBack depending on a lot of state/props
Maybe reorder params or add comment above to explain what each param is used for ?

Reply:
Comments added